plumbing/ps/publisher.go

```diff
@@ -0,0 +1,18 @@
+package ps
```

Comment:
a super thin wrapper like this makes me think we should consider not having it, instead just defining in the plumbing.api the publisher and subscriber interfaces we expect a ps dependency passed in to support and then to pass calls directly to that thing rather than through this wrapper